<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>爱表格</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTableWidget" name="tableWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="action_process"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>文档与帮助</string>
    </property>
    <addaction name="action_3"/>
    <addaction name="action_4"/>
    <addaction name="action_5"/>
    <addaction name="action_6"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>打开文件</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>导出文件</string>
   </property>
  </action>
  <action name="action_process">
   <property name="text">
    <string>载入插件</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>入门教程</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>插件中心</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>交流群</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>最新软件下载</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
